lib/grover/js/processor.cjs

```diff
@@ -172,6 +194,8 @@ const _processPage = (async (convertAction, urlOrHtml, options) => {
       await page.goto(displayUrl || 'http://example.com', requestOptions);
     }
 
+    handleErrors();
```

Comment:
I see that the opposite also needs to hold true. ie if the `wait_for_selector`/`wait_for_function` is waiting for some async data loading to complete and you want to have an error alerted if/when those have completed. This would be a breaking change for anyone expecting the current behaviour 😉 

It looks like if we checked for errors right away AND after the delayed checks it should resolve your use case whilst still allowing the behaviour to retain it's existing function.